components/StudentResource.jsx

```diff
@@ -7,24 +7,38 @@ import { Text } from "./Text";
 import { Button } from "./Button";
 import { media } from "../utils";
 
+/**
+ *
+ * Student Resource Component
+ * @prop {string} titleText - resource title
+ * @prop {string} descriptionText - resource description
+ * @prop {array} buttons - array of buttons with resource relevant links
+ * |- each button element should have text and link keys
+ * @prop {object} graphic - resource graphic
+ * @prop {number} graphicWidth - width of resource graphic, default 400
+ * @prop {number} graphicHeight - height of resource graphic, default 350
+ * @prop {boolean} isGraphicLeft - specify the location of the resource graphic
+ * |- default true (image on left, text on right)
+ * @prop {boolean} isCard - should the text + buttons look like a card, default false
+ * |- box shadow and rounded borders
+ */
+
 export const StudentResource = ({
```

Comment:
Almost feel like we should rename this component at this point lol

Reply:
Probably, just not sure what to rename it to. The LeftImageTextLayout one is similar, just has the title above the image. Any suggestions?